.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
 target
 *.iml
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+*.class
```

Comment:
Actually they werent ignored, in the beginning now even without the *.class at .gitignore its seems to be ignored, maybe its just an unstability from IDea :D, 
so i will just rollback this change :)